src/css/profile/wearable/changeable/common/spin.less

```diff
@@ -1,39 +1,48 @@
 .ui-spin {
-    position: relative;
-    display: inline-block;
-    height: 60 * @unit_base;
-    background-color: transparent;
-    overflow: hidden;
-    font-size: 22 * @unit_base;
-    padding: 0;
-    box-sizing: content-box;
-    width: 80 * @unit_base;
-    &-item {
-        background-color: transparent;
-        position: absolute;
-        left: 0;
-        top: 0;
-        width: 100%;
-        height: 30 * @unit_base;
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        color: rgba(77, 207, 255, 1);
+	position: relative;
+	display: inline-block;
+	height: 60 * @unit_base;
+	background-color: transparent;
+	overflow: hidden;
+	font-size: 22 * @unit_base;
+	padding: 0;
+	box-sizing: content-box;
+	width: 80 * @unit_base;
+	&-item {
+		background-color: transparent;
+		width: 100%;
+		height: 30 * @unit_base;
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		color: rgba(77, 207, 255, 1);
 
-        &-selected {
-            font-size: 30 * @unit_base;
-            height: 30 * @unit_base;
-            color: rgba(255, 255, 255, 1);
-        }
-    }
-    &-enabling {
-        height: 100%;
-        .ui-spin-item {
-            transition: 300ms opacity linear;
-        }
-    }
-    &-placeholder {
-        opacity: 0;
-        pointer-events: none;
-    }
+		&-selected {
+			font-size: 30 * @unit_base;
```

Comment:
use @sp_size;

Reply:
I cannot use `@sp_base` in wearable. I used `@px_base`